operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -980,7 +980,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
             alert: $.alertName('GoThreadsTooHigh'),
             expr: |||
               # We filter by the namespace because go_threads can be very high cardinality in a large organization.
-              max by(%(alert_aggregation_labels)s, %(per_instance_label)s) (go_threads{namespace=~".*(cortex|mimir).*"} > %(threshold)s)
+              max by(%(alert_aggregation_labels)s, %(per_instance_label)s) (go_threads{job=~".*(cortex|mimir).*"} > %(threshold)s)
```

Comment:
This will break the Jsonnet-based deployments, where workload names don't have a fixed prefix.

Reply:
> Then maybe we can just add baremetal to the namespace regex?

I don't think it will work. Baremetal deployments typically have no namespace.